orion.py

```diff
@@ -117,21 +117,13 @@ def get_metadata(test):
     Returns:
         dict: dictionary of the metadata
     """
-    metadata_columns = [
```

Comment:
I wonder if we keep some aspect of "known fields" and have Orion spit out if the user provides a field that is unknown?

Reply:
I just dont know if we should keep a list of columns if they are going to continue to change. Like how we added fips and etcd. Don't think we should constantly try to keep up to date with it